client/index.html

```diff
@@ -258,6 +258,12 @@ <h1 class="title">Settings</h1>
 									Show seconds in timestamp
 								</label>
 							</div>
+							<div class="col-sm-6">
+								<label class="opt">
+									<input type="checkbox" name="away">
+									Show away/back changes
```

Comment:
Should this be condensed when we do condensed messages? I reckon it should be just included like the rest of the join/part stuff.

Reply:
Condensed -- do we still want this option?

Reply:
I don't think so. This is a status message like any others, I think. So something like "X users went away, Y users came back" and 💥, part of the condensed stuff.

Reply:
Just noticed that's what you actually did so don't mind me 👍